# Let Fink install the document files into the proper location instead
# of having rdiff-backup's setup.py do it, which also puts its version
# number in the document directory name.

diff -ru rdiff-backup-1.1.0-orig/setup.py rdiff-backup-1.1.0/setup.py
--- rdiff-backup-1.1.0-orig/setup.py	2004-01-31 20:31:34.000000000 -0800
+++ rdiff-backup-1.1.0/setup.py	2004-03-08 19:01:56.000000000 -0800
@@ -56,7 +56,4 @@
 							   libraries=libname,
 							   extra_link_args=lflags_arg)],
 	  scripts = ['rdiff-backup'],
-	  data_files = [('share/man/man1', ['rdiff-backup.1']),
-					('share/doc/rdiff-backup-%s' % (version_string,),
-					 ['CHANGELOG', 'COPYING', 'README', 'FAQ.html'])])
-
+	  data_files = [('share/man/man1', ['rdiff-backup.1'])])

http://savannah.nongnu.org/cgi-bin/viewcvs/rdiff-backup/rdiff-backup/rdiff_backup/rpath.py.diff?r2=1.95&r1=1.94&diff_format=u
===================================================================
RCS file: /cvsroot/rdiff-backup/cvsroot/rdiff-backup/rdiff-backup/rdiff_backup/rpath.py,v
retrieving revision 1.94
retrieving revision 1.95
diff -u -r1.94 -r1.95
--- rdiff-backup/rdiff_backup/rpath.py	2005/11/06 01:46:54	1.94
+++ rdiff-backup/rdiff_backup/rpath.py	2005/11/06 06:51:17	1.95
@@ -1239,7 +1239,7 @@
 	from Carbon.File import FSSpec
 	import MacOS
 	try:
-		fsobj = FSSpec(self.path)
+		fsobj = FSSpec(rpath.path)
 		finderinfo = fsobj.FSpGetFInfo()
 		cfile = {'creator': finderinfo.Creator,
 				 'type': finderinfo.Type,
@@ -1248,7 +1248,7 @@
 		return cfile
 	except MacOS.Error:
 		log.Log("Cannot read carbonfile information from %s" %
-				(self.path,), 2)
+				(rpath.path,), 2)
 		return None
 
 
##
## http://savannah.nongnu.org/cgi-bin/viewcvs/rdiff-backup/rdiff-backup/rdiff_backup/log.py.diff?r2=1.19.2.1&r1=1.19&diff_format=u
##
===================================================================
RCS file: /cvsroot/rdiff-backup/cvsroot/rdiff-backup/rdiff-backup/rdiff_backup/log.py,v
retrieving revision 1.19
retrieving revision 1.19.2.1
diff -u -r1.19 -r1.19.2.1
--- rdiff-backup/rdiff_backup/log.py	2005/08/08 02:22:10	1.19
+++ rdiff-backup/rdiff_backup/log.py	2005/11/12 05:40:01	1.19.2.1
@@ -247,7 +247,7 @@
 
 	def write_if_open(cls, error_type, rp, exc):
 		"""Call cls.write(...) if error log open, only log otherwise"""
-		if not Globals.isbackup_writer:
+		if not Globals.isbackup_writer and Globals.backup_writer:
 			return Globals.backup_writer.log.ErrorLog.write_if_open(
 				error_type, rp, str(exc)) # convert exc bc of exc picking prob
 		if cls.isopen(): cls.write(error_type, rp, exc)
